src/main/kotlin/org/b12x/gfe/core/controller/version/LocalVersions.kt

```diff
@@ -0,0 +1,95 @@
+package org.b12x.gfe.core.controller.version
+
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import org.b12x.gfe.core.model.DataDownload
+import org.b12x.gfe.core.model.parsers.ParserVersionData
+import org.b12x.gfe.utilities.DirectoryManagement
+import org.b12x.gfe.utilities.FileManagement
+import java.io.File
+import java.nio.file.Paths
+import kotlin.io.path.exists
+
+class LocalVersions(loci: String) {
+
+    private val directoryManagement = DirectoryManagement()
+    private val gsgDataLocation = directoryManagement.setLociLocation(loci)
+    private val headerLength = 50  // bytes
+
+    val versionsList = createVersions()
```

Comment:
I don't see where this is used. If it's read from outside this class, shouldn't it be read-only?

Reply:
versionList isn't used yet because I haven't implemented it. They broke that chunk of the API so it's difficult to build around.